operators/StartMeterOperator.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+const MeterOperator = require('./meter-operator/MeterOperator');
```

Comment:
Where is this `MeterOperator` ? Could not find in this PR? Am I missing something?